src/math.ts

```diff
@@ -72,66 +72,39 @@ export function getWorldRotation(entity: Entity): Quaternion {
 /**
  * @public
  */
-export enum InterpolationType {
```

Comment:
mm i think this enum is used from the `@dcl/inspector` UI, if we remove it we will need a way to populate the dropdown

Reply:
yeah, the main change would be to update: 

https://github.com/decentraland/js-sdk-toolchain/blob/main/packages/%40dcl/inspector/src/components/EntityInspector/ActionInspector/TweenAction/TweenAction.tsx#L16-L33

so we have the proper labels for each value